README.md

```diff
@@ -1,2 +1,89 @@
 # pg-table_tests
 
+This project consists of a suite of tools to test and verify 5-level page table
+behavior on Linux.
+
+As documented in the [Linux kernel](Documentation/x86/x86_64/5level-paging.txt),
+5-level page tables boosts the virtual address space from 256 TiB
+(2<sup>47</sup> bytes) to 128 PiB (2<sup>57</sup> bytes).
+
+Since Linux splits its virtual address space between user and kernel, this
+means that applications can expect to see an increase from 128 TiB
```

Comment:
Since Linux splits its virtual address space between user and kernel, applications can....